src/common/options/global.yaml.in

```
@@ -2703,6 +2703,15 @@ options:
   services:
   - mon
   with_legacy: true
+- name: osd_pool_default_flag_ec_optimizations
+  type: bool
+  level: advanced
+  desc: enable performance optimizations for erasure coded pools
```

Let's rework this a bit. How about:

```suggestion
  desc: Control whether to create new erasure coded pools with EC optimizations turned on by default
```

---

Will do